### PR TITLE
fix: reorder navigation

### PR DESCRIPTION
**Pull request by vinckr**
- feat: header nav reorder
- fix: request access -> sign up

## Checklist

- [ ] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md)
      and signed the CLA.
- [ ] I have read the [security policy](../security/policy).
- [ ] I confirm that this pull request does not address a security
      vulnerability. If this pull request addresses a security vulnerability, I
      confirm that I got green light (please contact
      [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push
      the changes.
- [ ] I have added tests that prove my fix is effective or that my feature
      works.
- [ ] I have added necessary documentation within the code base (if
      appropriate).

### CODE REVIEW

File: src/page-content/navigation/navigation-header.tsx [hunk 1]
```diff
@@ -32,198 +34,211 @@ export const sideNav: Pick<NavProps, 'sideNav'> = {
...
+        {
+          iconLeft: Pricing,
+          title: 'Pricing',
+          description: "Ory Cloud Pricing",
+          to: '/pricing'
+        }
+      ]
+    },
+    {
+      title: 'Ory Open Source',
```
Comment (jfcurran) [inline src/page-content/navigation/navigation-header.tsx:62]:
```suggestion
      title: 'Open Source',
```
just call it open source?

File: src/page-content/navigation/navigation-header.tsx [hunk 2]
```diff
@@ -32,198 +34,211 @@ export const sideNav: Pick<NavProps, 'sideNav'> = {
...
+      mainMenu: [
         {
           iconLeft: OpenSource,
-          title: 'Open Source Ecosystem',
+          title: 'Open Source Overview',
```
Comment (jfcurran) [inline src/page-content/navigation/navigation-header.tsx:66]:
```suggestion
          title: 'Overview',
```

File: src/page-content/navigation/navigation-header.tsx [hunk 3]
```diff
@@ -32,198 +34,211 @@ export const sideNav: Pick<NavProps, 'sideNav'> = {
...
           },
           {
             title: 'Pricing',
-            to: '/pricing/'
+            to: '/pricing'
           },
+        ]
+      },
+      {
+        category: 'Ory Open Source',
```
Comment (jfcurran) [inline src/page-content/navigation/navigation-header.tsx:173]:
```suggestion
        category: 'Open Source',
```

File: src/page-content/navigation/navigation-header.tsx [hunk 4]
```diff
@@ -32,198 +34,211 @@ export const sideNav: Pick<NavProps, 'sideNav'> = {
...
+        mobileMenuItems: [
           {
             title: 'Open Source',
```
Comment (jfcurran) [inline src/page-content/navigation/navigation-header.tsx:176]:
```suggestion
            title: 'Overview',
```

**Review by jfcurran — CHANGES_REQUESTED** [review]
nice, I think we should make the wording for open source a bit easier here.
I also think jobs should be a top level item, what do you think?

**Comment by vinckr** [comment]
Maybe you can take a quick look at https://github.com/ory/web/pull/341/commits/103a57aea5d449844d59df35b1a0115ebb891c01 @jfcurran ? 

I changed the footer spacing, since it looked a bit crooked when there are 4 categories with the spacing before.

**Comment by jfcurran** [comment]
this looks a bit better in b48b0b2, what do you think? @vinckr 👀

**Comment by vinckr** [comment]
I tested it a bit on mobile, looking good 👍

**Review by jfcurran — APPROVED** [review]

**Comment by aeneasr** [comment]
Could we maybe find a different icon for pricing? 😅

File: src/page-content/navigation/navigation-header.tsx [hunk 5]
```diff
@@ -32,163 +34,176 @@ export const sideNav: Pick<NavProps, 'sideNav'> = {
...
   dropdownMenu: [
     {
-      title: 'Product',
+      title: 'Ory Cloud',
       mainMenu: [
         {
           iconLeft: Cloud,
-          title: 'Ory Cloud',
+          title: 'Sign Up',
           description: 'Never Build Auth Again',
```
Comment (aeneasr) [inline src/page-content/navigation/navigation-header.tsx:50]: Maybe something like "Become an Ory Insider"?

**Review by aeneasr — CHANGES_REQUESTED** [review]

**Comment by jfcurran** [comment]
@vinckr and myself pondered over wether it should be a coffee (buy us a coffee!) or a sack of money, or coins (something monetary). I believe both concepts could work here and I will recuse myself from this decision

File: src/page-content/navigation/navigation-header.tsx (hunk 5 above)
Comment (jfcurran) [reply src/page-content/navigation/navigation-header.tsx:50]: I agree - this needs to be more descriptive

Comment (jfcurran) [inline src/page-content/navigation/navigation-header.tsx:50]:
```suggestion
          description: 'Become an Ory Insider',
```

**Review by jfcurran — CHANGES_REQUESTED** [review]

**Comment by aeneasr** [comment]
I think a `$` would be the most indicative?

In general probably brings up the question if we really want icons for every menu item mid term

**Comment by jfcurran** [comment]
yes - truthfully, after having looked at it for a week, I am for reverting back to not using icons. Lets remove them in an upcoming styling patch for footer and nav - I am currently working on them anyway, and can remove the icons as well.

**Comment by aeneasr** [comment]
SGTM then we merge it as is and will revert to non-icons soon

**Pull request merged**